### ISSUE TITLE
On window zoom not working correctly

### ISSUE THREAD

**Issue by rikarsen** [rzajac/angularjs-slider#88]
hi.

I changed onMove function. Added eventX = event.clientX / window.Pagezoom; in if('clientX' in event) stantment

``
if('clientX' in event)
{
        eventX = event.clientX / window.Pagezoom;
}
``

**Comment by ValentinH** [comment]
Could you provided a Fiddle showing the problem you are solving with this code? You can fork this one: http://jsfiddle.net/pq7yr6d6/

**Comment by rikarsen** [comment]
http://jsfiddle.net/pq7yr6d6/1/ please drag and see what happened

**Comment by ValentinH** [comment]
I had the same problem when using `transform: scale(0.5)` in my css. I handled it by adding a `rz-slider-scale` parameter that do pretty much the same as you but also modifies the getWidth function to return the proper width.

Do you think it worth a Pull Request so that everybody can use it?

**Comment by rikarsen** [comment]
Yes i think. In new version it will be better if this issue is fixed.

**Comment by ValentinH** [comment]
Actually, using the zoom property isn't recommended: https://css-tricks.com/almanac/properties/z/zoom/ .
I have created a branch with my code for handling the scale property: it works with the transform property but not with the zoom. Indeed, the zoom property behaves weirdly since it modifies the event positions but not the sizes returned by `getBoundingClientRect()`.

Here's a fiddle using the scale parameter and a div resized through `transform: scale()`: http://jsfiddle.net/od2g4f9q/

**Comment by rikarsen** [comment]
thank you

**Comment by ValentinH** [comment]
Can I close this issue? Would you like the PR mentioned in this thread to be merged?

**Comment by elantion** [comment]
I use ```transform:translate()``` when I use ```ng-switch``` to switch dom element, this doesn't work too.

**Comment by ValentinH** [comment]
Could you please be mor eexplicit and provide an example through JSFiddle?

**Comment by mjcastillo** [comment]
I know this is an old post, and it looks like the scale option/feature did get into release 2.0.0, but it does not appear to work correctly. Just go to the demo page and you can reproduce the issue. I also saw your note on not really maintaining this anymore. Just wondering if anyone else out there found a way to make the scale feature work? Thanks for all the good work on this slider. nice job

**Comment by ValentinH** [comment]
Hey @mjcastillo , what's the issue?

I've just tested and everything looks fine to me: https://jsfiddle.net/ValentinH/ej5v8aru/

**Comment by mjcastillo** [comment]
i was baffled when I saw it work, then I saw what my issue was. The value that is used for the scale css transform function is the reciprocal of the value used for the scale option property! I thought the scale value in the options should be set to the same value as what I used in the css transform scale function, and now I see I was wrong. I need to use the reciprocal. Thank you very much for looking at this and responding. Have a great evening/day!
